backend/drive/drive.go

```diff
@@ -346,20 +345,6 @@ date is used.`,
 			Default:  "",
 			Help:     `Impersonate this user when using a service account.`,
 			Advanced: true,
-		}, {
-			Name:    "alternate_export",
```

Comment:
Removing this option completely will cause scripts with `--drive-alternate-export` in them to break.

What we should do here is leave it like this

```
                }, {
                        Name:     "alternate_export",
                        Default:  false,
                        Help:     "Deprecated: no longer needed",
                        Hide:     fs.OptionHideBoth,
                }, {
```

The last line hides it from the configurator and the command line flags, but it won't throw an error if used.